File: src/tests/Markify.Fixtures/Projects/Source/Interface/InterfaceSamples.cs
public interface FooType { }

public partial interface ParentType
{
    public partial interface NestedType { }
}

public partial interface ParentType
{
    public partial interface AnotherNestedType
    {
        public interface DeeperNestedType { }
    }
}

namespace FooNamespace
{
    public interface InNamespaceType { }
}

namespace FooNamespace.BarNamespace
{
    public partial interface ParentType
    {
        public interface NestedType { }
    }
}

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Interface/InterfaceSamples.vb
Public Interface FooType
End Interface

Public Partial Interface ParentType
    Public Partial Interface NestedType
    End Interface
End Interface

Public Partial Interface ParentType
    Public Partial Interface AnotherNestedType
        Public Interface DeeperNestedType
        End Interface
    End Interface
End Interface

Namespace FooNamespace
    Public Interface InNamespaceType
    End Interface
End Namespace

Namespace FooNamespace.BarNamespace
    Public Partial Interface ParentType
        Public Interface NestedType
        End Interface
    End Interface
End Namespace